frontend/ManagementController.cs

```diff
@@ -181,8 +174,14 @@ private AzureCredentials GetAzureCredentials()
             }
             else
             {
-                return factory.FromSystemAssignedManagedServiceIdentity(MSIResourceType.AppService,
-                                                                        AzureEnvironment.AzureGlobalCloud);
+                var tenantId = _config["SERVICE_PRINCIPAL_TENANT_ID"];

```

Comment:
Should keep the MSI logic here behind a flag or something once the bug is resolved?

Reply:
i wouldn't expect the app service bug to be fixed anytime soon. this current solution is fine for prod, i think (once we get secrets into KV, which is another TODO item)